frontend/__mocks__/hooks/api/runtimeData.ts

```diff
@@ -14,6 +14,7 @@ const mockedUseRuntimeData = useRuntimeData as jest.MockedFunction<
 export function getMockRuntimeDataWithPremium(): RuntimeData {
   return {
     FXA_ORIGIN: "https://example.com",
+    BASKET_ORIGIN: "https://example.com",
```

Comment:
suggestion (blocking): make this a separate value from the mock `FXA_ORIGIN` so if we're trying to debug it's more obvious what is what?
```suggestion
    FXA_ORIGIN: "https://fxa-mock.com",
    BASKET_ORIGIN: "https://basket-mock.com",
```

Reply:
Ah good one: 3d127b5c